pyiron/gaussian/gaussian.py

```diff
@@ -60,6 +63,60 @@ def from_hdf(self, hdf=None, group_name=None):
             self.input.from_hdf(hdf5_input)
             self.structure = Atoms().from_hdf(hdf5_input)
 
+    def visualize_MO(self,index):
+        n_MO = self.output.scf_density.shape[0]
+        assert index > 0 and index <= n_MO
+        assert len(self.output.numbers) < 50 # check whether structure does not become too large for interactive calculation of cube file
+
+        #make cube file
+        path = self.path+'_hdf5/'+self.name+'/input'
+        out = subprocess.check_output(
```

Comment:
This is very specific to the Gent cluster, so we should think about a way how we can generalise this option. Maybe the cube file can be generate at the end of every job?

Reply:
Both the MO's and the NMA functionality should be something that we have to generalise. 

For the MO part, I could not figure out how to load the complete cube file into nglview so this is what I came up with. However always creating the cube file is not very efficient, since we do not require it on a regular basis and it takes extremely long for extended systems (like MOF clusters). That is why I would prefer a post processing option for the cube file. Perhaps we could link it in some way to the gaussian executable?